pkg/controller/operators/labeller/filters.go

```diff
@@ -177,7 +178,12 @@ func Validate(ctx context.Context, logger *logrus.Logger, metadataClient metadat
 				logger.WithFields(logrus.Fields{
 					"gvr":           gvr.String(),
 					"nonconforming": count,
-				}).Info("found nonconforming items")
+				}).Errorf(
+					"found nonconforming items: missing the the required label %q (metadata.labels[\"%s\"]=\"true\"). "+
+						"Please label them appropriately, or consider deleting them if they are relate to a CRD from an uninstalled solution.",
```

Comment:
```suggestion
						"Please label them appropriately, or consider deleting them if they are related to a CRD from an uninstalled solution.",
```

Reply:
But I think @tmshort is saying, maybe, we don't want the whole corrective action part, the "Please..." part to show up in the logs.